index.html

```diff
@@ -583,42 +668,25 @@ <h2>Advisors</h2>
         </div>
         <div class="row">
 
-          <div class="col-lg-6">
-            <div class="member d-flex align-items-start" data-aos="zoom-in" data-aos-delay="100">
-              <div class="pic"><img src="assets/img/team/ag.jfif" class="img-fluid" alt=""></div>
-              <div class="member-info">
-                <h4>Anurag Goyal</h4>
-                <span>Advisor</span>
-                <p>B.Tech, Instrumentation Engineering, Class Of 2019</p>
-                <div class="social">
-                  <!-- <a href=""><i class="ri-twitter-fill"></i></a> -->
-                  <a href="https://www.facebook.com/anurag.goyal.1297"><i class="ri-facebook-fill"></i></a>
-                  <!-- <a href=""><i class="ri-instagram-fill"></i></a> -->
-                  <a href="https://www.linkedin.com/in/anurag-goyal-iitkgp/"> <i class="ri-linkedin-box-fill"></i> </a>
-                  <a href="javascript:;"><i class="ri-github-fill"></i></a>
-                </div>
-              </div>
-            </div>
-          </div>
-
-          <div class="col-lg-6 mt-4 mt-lg-0">
+          <div class="col-lg-6 mt-lg-0 mt-4">
             <div class="member d-flex align-items-start" data-aos="zoom-in" data-aos-delay="100">
-              <div class="pic"><img src="assets/img/team/jai.jpg" class="img-fluid" alt=""></div>
+              <div class="pic"><img src="assets/img/team/Somya_Patra.jpg" class="img-fluid" alt=""></div>
               <div class="member-info">
-                <h4>Jai Shukla</h4>
+                <h4>Soumya Patra</h4>
                 <span>Advisor</span>
```

Comment:
Soumya is the current Vice President of Prodex, not an advisor. Please make the required changes.